pandas/core/reshape/pivot.py

```diff
@@ -79,7 +79,7 @@ def pivot_table(data, values=None, index=None, columns=None, aggfunc='mean',
                 pass
         values = list(values)
 
-    grouped = data.groupby(keys)
+    grouped = data.groupby(keys, observed=dropna)
```

Comment:
Need to think about this a bit. Are we all OK with "overloading" `dropna` to serve two purposes? I think it's ok...

Reply:
yes, but that's the meaning of the ``dropna`` now here anyhow